sdk/core/core/test/cmocka/test_az_logging.c

```diff
@@ -201,3 +201,11 @@ void test_az_log(void** state)
   az_log_set_classifications(NULL);
   az_log_set_callback(NULL);
 }
+
+int test_az_logging()
```

Comment:
Is the goal to have the filename match this method name?

**Question:** Should we instead have the filename match the component it is testing? In this case, `az_log`?

I don't know if that's a better idea, but wanted to ask.

This comes up earlier as well where we are now merging az_http_response and az_http_request into one az_http file, but it now tests two separate types.

Reply:
naaa,, there is not any contract to match file name against functions.
that was just a pretty coincidence for this PR, XD

Might be helpful for someone who is not using F12 to go to functions definitions and want to use the file explorer instead.. But that´s like 1980´s 📦  :P